packages/babel-generator/test/fixtures/flow/interfaces-module-and-script/expected.js

```diff
@@ -1,9 +1,23 @@
-interface A {};
-interface A extends B {};
-interface A<T> extends B<T>, C<T> {};
-interface A { foo: () => number };
-interface Dictionary { length: number, [index: string]: string, };
+interface A {}
+;
```

Comment:
Are these actually parsed as `EmptyStatement` nodes, or is the semicolon being printed weirdly.